benchmark/compare.js

```diff
@@ -13,11 +14,12 @@ const cli = CLI(`usage: ./node compare.js [options] [--] <category> ...
   The output is formatted as csv, which can be processed using for
   example 'compare.R'.
 
-  --new    ./new-node-binary  new node binary (required)
-  --old    ./old-node-binary  old node binary (required)
-  --runs   30                 number of samples
-  --filter pattern            string to filter benchmark scripts
-  --set    variable=value     set benchmark variable (can be repeated)
+  --new      ./new-node-binary  new node binary (required)
+  --old      ./old-node-binary  old node binary (required)
+  --runs     30                 number of samples
+  --filter   pattern            string to filter benchmark scripts
+  --set      variable=value     set benchmark variable (can be repeated)
+  --progress true|false         show benchmark progress indicator
```

Comment:
The arg parser has some support, it defaults to `true` when no value is passed. However it will always expect the next argument to be the value, even if it starts with `--`. For example `--process --runs 20` will not work. The feature is not used anywhere, it is just an implementation detail.